routes/api.php

```diff
@@ -34,6 +34,8 @@
     Route::delete('/donations/{id}', [DonationController::class, 'destroy']);
     //petitions routes  
     Route::get('/petitions', [PetitionController::class, 'index']);
+    Route::get('/petitions/filter-month', [PetitionController::class, 'filterByMonth']);
+    Route::get('/petitions/filter-week', [PetitionController::class, 'filterByWeek']);
```

Comment:
It should be all at the same endpoint, like:

```
/petitions?time="week"
```

or

```
/petitions?time="month"
```

Reply:
Done